docs/src/pages/[platform]/getting-started/troubleshooting/troubleshooting.react.mdx

```diff
@@ -160,6 +160,19 @@ This is a known problem when using the `jsdom` library (a dependency of Jest) wi
 
 Next.js 13.4+ introduces [app directory](https://nextjs.org/docs/app/building-your-application/routing#the-app-directory) with the usage of [Server Components](https://nextjs.org/docs/getting-started/react-essentials#server-components). To use Amplify UI componenents, you must use them inside the Client Component tree by placing `"use client"` at the boundary between your Client and Server component module graph. Please see the Next.js [documentation](https://nextjs.org/docs/getting-started/react-essentials#third-party-packages) on how you could achieve this.
 
+### Next 13.4+: `Module not found` Errors
+
+When you use Amplify UI, you will see the follow errors from `aws-crt` and `encoding` in the server terminal:
```

Comment:
```suggestion
When you use Amplify UI with Server Components, you will see the follow errors from `aws-crt` and `encoding` in the server terminal:
```

Reply:
Might be worth considering changing "Amplify UI" to just "Amplify" here since Amplify UI itself is not the originator of the issue